src/ralph/admin/templates/admin/includes/javascripts.html

```diff
@@ -98,5 +98,26 @@
 
     (function($) {
         $(document).foundation();
+
+        $('li.has-dropdown').on('mouseover', function() {
+            var $this = $(this);
+            var parent = $(".top-bar");
+            var $thisdrop = $this.find(".dropdown");
+
+            var dropdowns = parent.find("li.has-dropdown .dropdown");
+            dropdowns.each(function() {
+                var $drop = $(this);
+                var width = 'auto';
+                if(!$drop.is($thisdrop)) {
+                    width = '';
+                    $drop.hide();
+                    setTimeout(function() {
+                        $drop.show();
+                    }, 300);
```

Comment:
btw, could we do sth like this: if the cursor is further from dropdown then visibility time decreases? :)